doc_src/tutorial.rst

```diff
@@ -649,6 +649,29 @@ This prompt would look like:
     :purple:`02/06/13`
     :red:`/home/tutorial >` _
 
+You can make your prompt transient by setting ``fish_transient_prompt`` to 1. That means fish will redraw the prompt before running a commandline, so you can change it then. Usually this is used to trim it down to declutter your terminal's scrollback history. When it runs the prompt again, it will pass a ``--final-rendering`` argument, so check for this to distinguish::
+
+    function fish_prompt
+        if contains -- --final-rendering $argv
```

Comment:
not sure if this should be part of the tutorial, which only contains very basic info.
We already have it on a separate "Writing your own prompt" page, we can maybe link to that from here.